lib/heap_sort.rb

```diff
@@ -1,8 +1,16 @@
-
+require_relative "min_heap"
 
```

Comment:
Just a note, that you **can** create a heap with linked nodes, but an array is more straightforward due to the fact that it's a complete tree.